app/assets/stylesheets/font_awesome.scss

```diff
@@ -0,0 +1,7 @@
+// https://stackoverflow.com/a/58087255
```

Comment:
The `fontawesome` documentation on getting this to work with webpack is quite lacking! But it didn't seem to be working with the gem version either, so 🤷 . This incantation appears to work, though i can't claim to understand what all is going on!